### DISCUSSION TITLE
Templates

### DISCUSSION THREAD

**Discussion by e-perl-NOAA** [nmfs-stock-synthesis/stock-synthesis#504] [General]
This is a discussion for SS3 users to ask for different stock synthesis input template files. Please subscribe to this discussion so that you get notifications about discussion posts.

**Comment by jules-selles** [comment]
Hey all, working on rock lobster and could use a template with a simple model when no age data are available. Thx

**Reply by MauroMardones** [reply]
I did used a simple template to a invertebrate (Loco) fishery in Chile with landings, index (survey), length compositions and LH parameters. It's not clear to me what data sources you are using. Anyway, take a look to see if they help you.

[template no age data](https://github.com/MauroMardones/SA_SS3_Loco)

**Comment by Rick-Methot-NOAA** [comment]
Thanks for sharing this.  We are actively thinking about how to provide templates for more situations.
